codalab/client/local_bundle_client.py

```diff
@@ -274,54 +274,11 @@ def _bundle_inherit_workheet_permissions(self, bundle_uuid, worksheet_uuid):
 
     @authentication_required
     def kill_bundles(self, bundle_uuids):
-        """
-        Send a kill command to all the given bundles.
-        """
-        check_bundles_have_all_permission(self.model, self._current_user(), bundle_uuids)
-        for bundle_uuid in bundle_uuids:
-            worker_message = {
-                'type': 'kill',
-                'uuid': bundle_uuid,
-            }
-            action_string = BundleAction.kill()
-            self._do_bundle_action(bundle_uuid, worker_message, action_string)
+        raise UsageError("API Updated: Please upgrade to the latest version of the CLI.")
```

Comment:
Do we need this?  Wouldn't it be better to have a generic way whereby a server can tell the client the version number, and if this is higher than the client, then it prints out a warning, and recommend the client to upgrade.